ci-operator/step-registry/assisted/verify-generated-code-v2/assisted-verify-generated-code-v2-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: assisted-verify-generated-code-v2
+  from: assisted-service-generate
```

Comment:
the only difference between ``assisted-verify-generated-code`` and ``assisted-verify-generated-code-v2`` is this line of code?
because if so we don't need to duplicate this step. As long as the image does the same thing we can do (in ``release-ocm-2.5`` configuration file):
```
images:
...
- dockerfile_literal: |
    FROM base
  from: assisted-service-build
  to: assisted-service-generate
```
and the step will run with no problems